pkg/minikube/cluster/cluster.go

```diff
@@ -20,6 +20,7 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
+	"github.com/docker/machine/libmachine/ssh"
```

Comment:
this change looks odd

Reply:
Agree. That was definitely added in error. I'll fix.